### DISCUSSION TITLE
How to modify the code to move the location of the `assets` directory?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#280] [General]
This is needed to install ecode into the file system in Linux. According to the Linux FHS, the location would be `/usr/share/ecode/assets`.

**Comment by SpartanJ** [comment]
You just need to patch the resources path in [ecode.cpp](https://github.com/SpartanJ/eepp/blob/develop/src/tools/ecode/ecode.cpp#L3421C1-L3422C37). So basically change it to `mResPath = "/usr/share/ecode/"` (and a few lines below will append `assets` to it).
